charts/rode-collector-image-scanner/templates/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "rode-collector-image-scanner.fullname" . }}
+  labels:
+    {{- include "rode-collector-image-scanner.labels" . | nindent 4 }}
+spec:
+{{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+{{- end }}
+  selector:
+    matchLabels:
+      {{- include "rode-collector-image-scanner.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "rode-collector-image-scanner.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default (printf "v%s" ($.Chart.AppVersion) ) }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - "--port={{ .Values.service.port }}"
+            - "--rode-host={{ .Values.rode.host }}"
+            {{- if .Values.rode.auth.proxy.enabled }}
+            - "--proxy-auth"
+            {{- end }}
+            - "--rode-insecure-disable-transport-security={{ .Values.rode.disableTransportSecurity }}"
+            - "--debug={{ .Values.debug }}"
+          ports:
+            - name: http
+              containerPort: {{ .Values.service.port }}
+              protocol: TCP
+          readinessProbe:
+            exec:
+              command: ["/grpc_health_probe", "-addr=:{{ .Values.service.port }}"]
+            initialDelaySeconds: 5
+          livenessProbe:
+            exec:
+              command: ["/grpc_health_probe", "-addr=:{{ .Values.service.port }}"]
+            initialDelaySeconds: 5
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+        - name: trivy-server
```

Comment:
It looks like the server has a `/healthz` endpoint if we want to use that for probes: https://github.com/aquasecurity/trivy/blob/3b6122f86db6d919b629318865d6760c594ff74b/pkg/rpc/server/listen.go#L98